simulation_parameters/microbe_stage/organelles.json

```diff
@@ -1083,6 +1084,57 @@
     "EditorButtonGroup": "Organelle",
     "EditorButtonOrder": 5
   },
+  "ferroplast": {
+    "MPCost": 45,
+    "InitialComposition": {
+      "ammonia": 2,
+      "phosphates": 2
+    },
+    "Hexes": [
+      {
+        "r": 0,
+        "q": 0
+      },
+      {
+        "r": -1,
+        "q": 0
+      }
+    ],
+    "Processes": {
+      "ferrosynthesis": 1
+    },
+    "Components": {
+      "Storage": {
+        "Capacity": 1
+      }
+    },
+    "Enzymes": {
+      "rusticyanin": 1
+    },
+    "ProkaryoteChance": 0,
+    "ChanceToCreate": 1,
+    "Density": 1100,
+    "Graphics": {
+      "ScenePath": "res://assets/models/organelles/Ferroplast.tscn"
+    },
+    "Name": "FERROPLAST",
+    "ProductionColour": "#3293f7",
+    "ConsumptionColour": "#f04681",
+    "IconPath": "res://assets/textures/gui/bevel/parts/FerroplastIcon.png",
+    "UnlockConditions": [
+      {
+        "ReproduceWithOrganelle": {
+          "Organelle": "rusticyanin",
+          "Generations": 7,
+          "InARow": true
+        }
+      }
+    ],
+    "RequiresNucleus": true,
+    "EditorButtonGroup": "Organelle",
+    "EditorButtonOrder": 5,
```

Comment:
Editor button orders should be unique within the group. It looks like 5 is already used by lysosome in the organelle group. This is a requirement to ensure that the sort order is fully consistent in the editor GUI.